profiles/lib/directional.lua

```diff
@@ -53,6 +53,13 @@ function Directional.get_values_by_set(way,data,keys)
     end
   end
 
+  -- allow access in undefined in one direction and defined in opposite one
```

Comment:
I think this implies usage in the access restriction context. This function make no assumptions about tag values however. This would need to be handled in the caller code.